packages/relay/README.md

```diff
@@ -127,25 +127,38 @@ The `sendTransaction` call returns once the transaction has been _signed_ by the
 
 #### Speed
 
-Instead of accepting a fixed `gasPrice`, the relayer accepts a `speed` parameter that can be one of `safeLow`, `average`, `fast`, or `fastest`. The relayer will determine the gas price based on these values from [ethgasstation](https://ethgasstation.info/), and update it regularly by resubmitting your transaction if it fails to get mined.
+Instead of accepting a fixed `gasPrice` or `maxFeePerGas/maxPriorityFeePerGas`, the relayer accepts a `speed` parameter that can be one of `safeLow`, `average`, `fast`, or `fastest`. The relayer will determine the gas price based on values from different oracles such as [ethgasstation](https://ethgasstation.info/), and update it regularly by resubmitting your transaction if it fails to get mined.
```

Comment:
nitpick, but I would just include the param names in the ticks and not the slash:

```suggestion
`maxFeePerGas`/`maxPriorityFeePerGas`
```

Reply:
This line makes it sound like the only thing a Relayer will accept is `speed`:

> Instead of accepting a fixed `gasPrice` or `maxFeePerGas/maxPriorityFeePerGas`, the relayer accepts a `speed` parameter

But if I'm reading the code right, it seems like it would be valid to provide one and only one of:

* speed
* maxFeePerGas/maxPriorityFeePerGas
* gasPrice

Reply:
You're right, I updated with both suggestions, thanks!